src/components/replay.js

```diff
@@ -0,0 +1,24 @@
+AFRAME.registerComponent("replay", {
+  init: function() {
+    this.playhead = 0;
+    this.poseIndex = 0;
+  },
+
+  tick: function(t, dt) {
+    let overflow = false;
+    while (!overflow && this.playhead >= this.poses[this.poseIndex].timestamp) {
+      this.el.setAttribute("position", this.poses[this.poseIndex].position);
+      this.el.setAttribute("rotation", this.poses[this.poseIndex].rotation);
+      this.el.object3D.updateMatrix();
```

Comment:
Probably not a huge deal since its just bots, but why not just set the flag here?

Reply:
No reason besides that I copied the code